### ISSUE TITLE
files in .git/info/exclude ignored whatever "Exclude VCS Ignored path"

### ISSUE THREAD

**Issue by colljori** [atom/atom#16426]
### Prerequisites

* [x] 
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Problem about the .git/info/exclude file. Files and repertory specified in it are ignored by all packages, even if the "Exclude VCS Ignored path" option is uncheck.

### Steps to Reproduce

1. Clone a project from a git repo, this project must have a exclude file in .git/info
[I can give you one if needed]
2. Unvalid the "Exclude VCS Ignored path" option in core

**Expected behavior:** 
Packages such as "fuzzy-finder" or "search in project" should not exclude the files excluded by git specified in .git/info/exclude
Also these files and repertories should not be grayed in the tree-view.

**Actual behavior:** 
All files and repertories specified in .git/info/exclude are ignored by these packages.
Also these files and repertory are grayed in the tree-view.

**Reproduces how often:** 
100%

### Versions

Atom    : 1.23.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0        
apm  1.18.11
npm  3.10.10
node 6.9.5 x64
python 2.7.9
git 2.1.4

**Comment by rsese** [comment]
Thanks for reaching out - do you have `Settings > Packages > tree-view > Hide VCS Ignored Files` unchecked?

**Comment by colljori** [comment]
Thanks for reply!

Yes the `Settings > Packages > tree-view > Hide VCS Ignored` is unchecked.
And I precise that file aren't hide in the tree-view, but only grayed.

**Comment by rsese** [comment]
> Clone a project from a git repo, this project must have a exclude file in .git/info
> [I can give you one if needed]

That would be great if you can share a test repo that I can try to reproduce with 👍

**Comment by colljori** [comment]
Here an exemple https://github.com/colljori/atomIssueExemple

Unfortunately only a small part of the issue is show here.
You can see the thisShouldntBeIgnored file grayed in the tree view, either the Ignored VCS path (core) option is check or not.

I don't manage to reproduce the others issue on this simple exemple. 

Edit: toggling  Ignored VCS path (core) option need a reboot of atom for package such as fuzzy finder or search in project to work.

**Comment by rsese** [comment]
> You can see the thisShouldntBeIgnored file grayed in the tree view, either the Ignored VCS path (core) option is check or not.

Yes I believe this is expected behavior if I'm understanding you correctly?  In the tree view, the `Settings > Packages > tree-view > Hide VCS Ignored Files` setting will completely hide the file in the tree view if checked.  If not checked, the file won't be hidden but it will still have a slight grey color applied to it to indicate that it's an ignored file.  Are you describing this or something else?
